tests/unit/concrete/batch/batch.tree

```diff
@@ -0,0 +1,24 @@
+Batch_Unit_Concrete_Test
+├── when function does not exist
+│  └── it should revert
+└── when function exists
+   ├── given non state changing function
```

Comment:
this should be a `when`

All the other branches refer to properties of the same function.

Reply:
Good point.